### ISSUE TITLE
Add Tests

### ISSUE THREAD

**Issue by hoffi** [jansmolders86/mediacenterjs#190]
Add Tests to the project to ensure things like Title-Cleaner for example don't get broken.

**Comment by hoffi** [comment]
I've already created a branch `tests` where i have added a first test for the `title-cleaner` and give a set of bad filenames and then expect specific results

**Comment by jansmolders86** [comment]
Ow wauw this is quite awesome! you use mocha to run these tests I see?

**Comment by hoffi** [comment]
Yes i have already used Mocha in my other projects and i like it :)

**Comment by drBenway** [comment]
maybe add something like:
"test": "node node_modules/mocha/bin/mocha spec/**/*.spec.js"
to the package.json file so we can just npm run test and trigger Mocha
